### ISSUE TITLE
RC_v1.5.0 API docs broken

### ISSUE THREAD

**Issue by freemansw1** [tobac-project/tobac#297]
https://tobac.readthedocs.io/en/rc_v1.5.0/tobac.html

**Comment by w-k-jones** [comment]
Just went back through the pull requests for `RC_v1.5.0`, the docs breaking occurs with #259

**Comment by fsenf** [comment]
autodoc seems to run into an import error:
```
...
WARNING: autodoc: failed to import module 'analysis' from module 'tobac'; the following exception was raised:
...
```

**Comment by fsenf** [comment]
OK, now I read further:
```
...
  File "/home/docs/checkouts/readthedocs.org/user_builds/tobac/checkouts/293/tobac/utils/periodic_boundaries.py", line 261, in <module>
    high: float = 2 * np.pi,
TypeError: unsupported operand type(s) for *: 'int' and '_MockObject'
```

The problem comes with the fact that we mock all python libs like numpy and this appear to be in conflict with the variable definition using type hints. 

@freemansw1 : You could try out `float = 2. * np.pi` ...

**Comment by JuliaKukulies** [comment]
I tried that @fsenf @freemansw1 and the API looks fine when I locally build the page, but it does not seem to do the job (#303 ). 

The pull requests that have not merged in the period boundary changes seem to work: 
https://tobac--281.org.readthedocs.build/en/281/tobac.html#tobac-utils-general-modules  

And we had already changed the `utils` structure there, so I am not sure if it is really related to the `utils` submodules @w-k-jones. We forgot indeed to add `tobac.utils.periodic_boundaries` and `tobac.utils.internal` to `doc/tobac.rst` but this does also not fix it

**Comment by fsenf** [comment]
Hi,

I was able to reproduce the bug in a clean setup using the description here: https://github.com/tobac-project/tobac-tutorials/blob/main/docs/Testing-Sphinx-based-Rendering.md up to the end of Section 1 (replacing tobac-tutorials with tobac).

Some additional commands:
```bash
 pip install sphinx_rtd_theme
sphinx-build -b html doc doc/_build/html
```

Last command also fails with
```
...
  File "/tmp/website-testing/tobac/tobac/utils/periodic_boundaries.py", line 261, in <module>
    high: float = 2 * np.pi,
TypeError: unsupported operand type(s) for *: 'int' and 'pi'
...
```

**Comment by fsenf** [comment]
`float = 2. * np.pi` does not help.

**Comment by fsenf** [comment]
@freemansw1 @w-k-jones @JuliaKukulies : Can we afford to install numpy at each readthedocs re-rendering? I think, yes. What do you think?

**Comment by w-k-jones** [comment]
Would it be possible to switch to using `math.pi` instead?

**Comment by freemansw1** [comment]
Resolved with #305
